src/types/numeric.md

```diff
@@ -59,9 +59,36 @@ r[type.numeric.int.size.minimum]
 > pointer support is limited and may require explicit care and acknowledgment
 > from a library to support.
 
-## Bit validity
+## Representation
 
-r[type.numeric.validity]
+r[type.numeric.repr]
 
-For every numeric type, `T`, the bit validity of `T` is equivalent to the bit
-validity of `[u8; size_of::<T>()]`. An uninitialized byte is not a valid `u8`.
+r[type.numeric.repr.integer]
+Each value of an integer type is a whole number. For unsigned types, this is a positive integer or `0`. For signed types, this can either be a positive integer, negative integer, or `0`.
+
+r[type.numeric.repr.integer-width]
+The range of values an integer type can represent depends on its signedness and its width, in bits. The width of type `uN` or `iN` is `N`. The width of type `usize` or `isize` is the value of the `target_pointer_width` property.
```

Comment:
Shouldn't this say *how* it depends on that?

Specifically, unsigned integers carry values in `0 .. 2^N`, and signed integers carry values in `-2^(N-1) .. 2^(N-1)` (left-inclusive, right-exclusive).